### PR TITLE
added a demo video

### PR DESCRIPTION
**Pull request by suhasasumukh**
Hey there!

I have added a demo video from your twitter handle to the readme file so that users can easily understand how to use this cloner

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,3 +16,8 @@ REPLICATE_API_TOKEN=<token>
 npm install
 npm run dev
 ```
+
+## Demo Video
+
+<https://github.com/suhasasumukh/voice-cloner/assets/70210383/2fd79d23-f295-4354-a07e-ea094c93e26b>
```
Comment (Pwntus) [inline README.md]: Please make this point to the file in this repository. Also, use the following format to get the thumbnail included:
```
[![Demo](./readme-demo/thumbnail.png)](./readme-demo/demo.mp4 "Demo")
```

File: readme-demo/thumbnil.png
Comment (Pwntus) [inline readme-demo/thumbnil.png:1]: Please rename to "thumbnail.png".

**Review by Pwntus — CHANGES_REQUESTED** [review]
Thanks! I've left some comments on your proposed additions.

File: README.md (hunk 1 above)
Comment (suhasasumukh) [reply README.md]: sure i will add this and fix the typo

**Comment by suhasasumukh** [comment]
I have fixed the changes you have asked me, please do check it :)

**Comment by Pwntus** [comment]
Thank you!

**Pull request merged**